content/information-distribution/source.md

```diff
@@ -75,6 +76,33 @@ In such an event, client implementations **should** consider articles posted
 prior to the key being declared as compromised as probably not having been
 tampered with, with no way to be 100% sure about it.
 
+## Change of Matrix user
+
+A source or subsource might have to change the Matrix user it uses to publish
+information at some points in its life time. This can happen for many reason:
+the homeserver it was registered on went down, or got compromised, or got
+isolated from the rest of the federation...
+
+In such an event, the source or subsource **must** publish a new registration
+event from its new Matrix user with the `prev_id` key having its previous user's
+ID as its value.
+
+This key **must** be used by client implementations to link the previous user to
+the new one, considering both as the same entity. A missing `prev_id` key or an
+empty string simply means that the source or subsource didn't previously use
+another Matrix account to publish information.
+
+If the entity is a source (and not a subsource), trust authorities certifying it
+as trustworthy **must** update their list of trusted entities by removing the
+previous user from it, and adding the new one to it. The signature associated
+with the new user **must** be generated from the new user's registration event.
+Client implementations **should** consider a trust authority not updating its
+list with the source's new user as if it stopped trusting it.
```

Comment:
I think this is hard to enforce and potentially dangerous: If an attacker registers deceptive source with a prev_id pointing to a legit source, TAs must not update their list with the new deceptive users, and the legit source should still be trusted as if nothing happened.

The only way I see for a TA to detect if a source has changed its user, is by contacting or being contacted by the source with other communication tools.